### PR TITLE
Lists

### PR DESCRIPTION
**Pull request by michaelquinn32**
This is an internal refactor of skim_to_lists, along with some small doc changes.

### CODE REVIEW

File: R/skim.R [hunk 1]
```diff
@@ -67,8 +67,10 @@ skim.default <-function(.data, ...){
   structure(skimmed, class = c("skim_vector", class(skimmed)) )
 }
 
-#' Print useful summary statistic from a data frame returning the data frame
-#'  without modification
+
+#' Skim a data frame without modification
```
Comment (elinw) [inline R/skim.R]: I actually think this is confusing grammatically which is why I changed it to the wordier version. Open to suggestions though. "Skim a data frame and return it without modification" maybe.   Right now the "without modification" modifies the "data frame" being skimmed.  What's important is that the original data frame is returned and the skim results are always printed.

File: R/skim.R [hunk 2]
```diff
@@ -104,35 +109,37 @@ skim_to_wide <- function(x, ...) {
...
+#' # Save the result
+#' sl <- iris %>% skim_to_list() 
+#' sl[["numeric"]]
+#' kable(sl$numeric)
+#' 
+#' # Or grouped, using magrittr exposition pipe
+#' # see ?magrittr::`%$%`
+#' iris %>%
+#'   group_by(Species) %>%
+#'   skim_to_list() %$%
```
Comment (elinw) [inline R/skim.R]: Typo should be %>%.  That's why the test is failing.

Comment (elinw) [reply R/skim.R]: `CO2 %>% dplyr::group_by(Type) %>% skim_to_list() `   works

File: R/skim.R (hunk 1 above)
Comment (michaelquinn32) [reply R/skim.R]: We can go back, but the first sentence is inappropriately long for a documentation title and gets recopied as a doc description:

https://www.rdocumentation.org/packages/skimr/versions/1.0.1/topics/skim_tee

What makes sense for hitting both goals (i.e. short title and clear description) together?

**Comment by elinw** [comment]
It’s definitely too long, don’t know what I was thinking with that …

**Pull request merged**